.gitignore

```diff
@@ -4,3 +4,5 @@ cmd/wboxd/wboxd
 cmd/wboxd/wboxd.toml
 cmd/wbox/wbox
 cmd/wbox/wbox.toml
+vendor/
```

Comment:
Using `.git/info/exclude` for system/editor-specific is generally a good idea.
Could you remove these?

Reply:
Sure!